opennms-doc/guide-user/src/asciidoc/text/service-assurance/introduction.adoc

```diff
@@ -0,0 +1,53 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+This section will cover the basic functionalities how _OpenNMS_ monitors availability and latency from applications or management agents.
+To change the behavior of how _OpenNMS_ monitors applications or status information from management agents please see _Administration Guide_.
+To extend the _Service Monitor_ framework please see _Development Guide_ of the _OpenNMS_ documentation.
```

Comment:
see *the* Development Guide

'of the OpenNMS documentation' can be omitted here.